.eslintrc.json

```diff
@@ -16,9 +12,30 @@
     "ecmaVersion": "latest",
     "sourceType": "module"
   },
-  "plugins": ["react", "@typescript-eslint"],
+  "plugins": ["react", "@typescript-eslint", "import-newlines", "newline-destructuring"],
   "rules": {
     "react/react-in-jsx-scope": "off",
-   "react/jsx-filename-extension": [1, { "extensions": [".ts", ".tsx"] }]
+    "react/jsx-filename-extension": [1, { "extensions": [".ts", ".tsx"] }],
+    "indent": ["error", 2],
+    "object-curly-newline": [
+      "error",
+      {
+        "ObjectExpression": { "minProperties": 2 },
+        "ObjectPattern": { "minProperties": 2 },
+        "ImportDeclaration": { "minProperties": 5 },
+        "ExportDeclaration": { "minProperties": 2 }
+      }
+    ],
+    "import-newlines/enforce": "warn",
+    "newline-destructuring/newline": "warn",
+    "no-duplicate-imports": "warn",
+    "no-console": "warn",
+    "prefer-const": "warn",
+    "no-var": "off"
+  },
+  "settings": {
+    "react": {
+      "version": "detect"
+    }
```

Comment:
nice work @camilovegag.

#### this is a very subjective opinion:

I think eslint should not care about formatting. Eslint should check for code quality and prettier should take care of this. However, i know that the plugins that were added to this config are not available as prettier config, so i suggest we scrap that from the requirement that we have for formatting.

my ideal settings for eslint:
```json
{
  "env": {
    "browser": true,
    "es2021": true
  },
  "extends": ["eslint:recommended", "plugin:react/recommended", "plugin:@typescript-eslint/recommended"],
  "parser": "@typescript-eslint/parser",
  "parserOptions": {
    "ecmaFeatures": {
      "jsx": true
    },
    "ecmaVersion": "latest",
    "sourceType": "module"
  },
  "plugins": ["react", "@typescript-eslint", "prettier"],
  "rules": {
    "react/react-in-jsx-scope": "off",
    "react/jsx-filename-extension": [1, { "extensions": [".ts", ".tsx"] }],
    "no-console": "warn",
    "no-duplicate-imports": "warn",
    "prefer-const": "warn"
  }  
}

```

cc @mjadach-iv